tests/ecs-service/index.test.ts

```diff
@@ -27,7 +27,11 @@ const programArgs: InlineProgramArgs = {
 };
 
 describe('EcsService component deployment', () => {
-  const region = process.env.AWS_REGION || 'us-east-2';
+  const region = process.env.AWS_REGION;
```

Comment:
What is the reason behind removing the default?

Reply:
My comment in the first code review. 👀